en/1.11.0/guide/scripts/index.md

```diff
@@ -0,0 +1,109 @@
+---
+layout: page
+title: Scripts
+---
+
+This guide will teach you how our crops' scripts work and how to make a good script,
+then you will be able to take the best of APISENSE.
+
+# Script execution philosophy
+
+When you write a script on APISENSE, you will have to take into account that this script will be executed only one time by our Bees, then all the data collection will be done thanks to events.
+Thus you may prefer events and repetitive actions to one shot behaviour. We will present some simple cases in the following sections.
+
+## One shot actions
+
+First of all, this is absolutely possible to write a script that will only execute actions at the moment of its interpretation, then do nothing.
+
+For example, the following script will save all data regarding the battery, then upload them one time, at the script startup:
+
+    var battery = require('battery');
+    var recorder = require('recorder');
+
+    recorder.save(battery.data()); // The data() method will create a json with a snapshot of the available data from our battery Sting.
+    recorder.sync();
+
+## Callbacks on events
+
+You may want to upload data regularly, depending on events that the Bees' phone encountered. For this you can register _callbacks_ on predefined events.
+
+For instance this script will __not__ upload data on startup but record data when a change in the battery connection state is detected:
+
+    var battery = require('battery');
+    var recorder = require('recorder');
+
+    battery.onStateChanged(function(data) {
+        recorder.save(data); // The data object here is a snapshot of the battery data when the event has been triggered
+        recorder.sync();
+    });
+
+
+## Repetitive behaviour
+
+I you don't want to be dependant on events, but want a regular exeuction of your data collection, you can also use a `timer` Sting to create repetitive actions.
```

Comment:
- Minor typo: s/I you don't/I**f** you don't
- What about giving a link to the Timer Sting documentation?